frontend/components/Dashboard/Profile/Profile.tsx

```diff
@@ -6,14 +6,14 @@ import { User as UserType } from '../../../generated/graphql'
 import theme from '../../../theme'
 
 type Props = {
-  currentUser: UserType
+  user: UserType | any
```

Comment:
Is the `any` because `userById` has extra language stuff on it that `UserType` doesn't have?

Reply:
Still not 100% sure, I tried a whole bunch of things but was having a lot of trouble with it and I saw that we had to do the same thing with Posts

Reply:
I tried also using `UserWithLanguagesFragmentFragment as UserType` for example but yeah nothing that I tried worked

Reply:
Going to merge for now since you approved as I'm working on some stuff that needs this, but if you end up having any more thoughts on it we can update this in my follow-up! 🙂